tools/privacyidea-diag

```diff
@@ -56,6 +56,16 @@ case $OS in
 		;;
 esac
 
+check_root() {
+	log "SECTION: Check Root"
+	log "==================="
+  if [ "$(id -u)" == 0 ]; then
+    log "This script is running as root."
+  else
+    log "This script is not running as root."
+  fi
+}
+
```

Comment:
i was thinking more of something like
```
if [[ $(id -u) != 0 ]]; then
  echo "Script must be run as root user"
  exit 1
fi
```
right at the beginning of the script.

Reply:
fixed in 808035780551993c88b00257271d96f89bbac166